Assets/Scripts/GameManager/GameManager.UI.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public partial class GameManager
+{
+    public string getCompanyName()
+    {
+        return companyName;
+    }
```

Comment:
Dit wordt in C# meestal op een andere manier gedaan. Je kan eens fullprop opzoeken if you want.
```suggestion
    public string CompanyName
    {
        get
        {
            return companyName;
        }
    }
```
Dit is de meest uitgebreide versie hiervan. Als er voor de rest geen andere checks gaan komen kan je die schrijven als hieronder.
```
public string CompanyName
{
    get => companyName;
}
```

Of, als het volledig gewoon een public getter is kan je het zo schrijven:
```
public string CompanyName => companyName;
```

Nog 1 last note, die private methods worden dan meestal geschreven met een underscore voor de naam. Dit zorgt er dan voor dat het verschil tussen de private var en de public getter/setter nog duidelijker is.